answer-transcribe-start.py

```diff
@@ -119,22 +117,20 @@ def handler(event, context):
     for record in event["Records"]:
         body = json.loads(str(record["body"]))
         request = json.loads(str(body["Message"]))["request"]
-        task_list = request["task_list"]
-        tasks = list(filter(lambda t: t["task_name"] == "transcribing", task_list))
-        if not tasks:
+        task = request["transcribeTask"] if "transcribeTask" in request else None
```

Comment:
you should probably update the transcribe sample event then

Reply:
answer-transcribe-start also uses answer-event.json.dist which contains the transcribeTask
answer-transcribe-collect uses transcribe-collect-event.json.dist which is already setup correctly